### PR TITLE
fix: UAT and regression v4.27.0

### PR DESCRIPTION
**Pull request by banklesss**
Related to:

- [YOMO-1339](https://emurgo.atlassian.net/browse/YOMO-1339)
- [YOMO-1338](https://emurgo.atlassian.net/browse/YOMO-1338)
- [YOMO-1343](https://emurgo.atlassian.net/browse/YOMO-1343)

[YOMO-1339]: https://emurgo.atlassian.net/browse/YOMO-1339?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[YOMO-1338]: https://emurgo.atlassian.net/browse/YOMO-1338?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[YOMO-1343]: https://emurgo.atlassian.net/browse/YOMO-1343?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/wallet-mobile/src/Dashboard/EpochProgress.tsx
Comment (banklesss) [inline apps/wallet-mobile/src/Dashboard/EpochProgress.tsx:1]: https://emurgo.atlassian.net/browse/YOMO-1339

File: apps/wallet-mobile/src/features/Settings/ChangeWalletName/ChangeWalletName.tsx
Comment (banklesss) [inline apps/wallet-mobile/src/features/Settings/ChangeWalletName/ChangeWalletName.tsx:1]: https://emurgo.atlassian.net/browse/YOMO-1338

File: apps/wallet-mobile/src/features/Settings/RemoveWallet/RemoveWalletScreen.tsx
Comment (banklesss) [inline apps/wallet-mobile/src/features/Settings/RemoveWallet/RemoveWalletScreen.tsx:1]: https://emurgo.atlassian.net/browse/YOMO-1343

File: apps/wallet-mobile/src/features/Settings/RemoveWallet/RemoveWalletScreen.tsx [hunk 1]
```diff
@@ -66,7 +66,9 @@ export const RemoveWalletScreen = () => {
           </WalletInfo>
         </ScrollView>
 
-        <Actions>
+        <Spacer fill />
+
+        <View style={styles.actions}>
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Settings/RemoveWallet/RemoveWalletScreen.tsx]: can put back <Actions? (less overhead for readability)

**Review by jorbuedo — APPROVED** [review]

**Review by michalszorad — APPROVED** [review]

File: apps/wallet-mobile/src/features/Settings/RemoveWallet/RemoveWalletScreen.tsx (hunk 1 above)
Comment (banklesss) [reply apps/wallet-mobile/src/features/Settings/RemoveWallet/RemoveWalletScreen.tsx]: done

**Pull request merged**